_templates/template.md

```diff
@@ -1,106 +1,228 @@
 ---
```

Comment:
Guys, you rewrote the template. This needs to be reverted to the original version.

CC: @nevalsar